.github/workflows/acceptance.yml

```diff
@@ -92,7 +92,7 @@ jobs:
       # Until GH composite actions can use `uses`, we need to setup python here
       - uses: actions/setup-python@v2
         with:
-          python-version: 2.7.17
+          python-version: 3.6.10
```

Comment:
Can we do what we are doing in `acceptance-py3.6.yml` where it reads from `.python-version` file?

Reply:
Addressed.